conf/test/llama3_70b.toml

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name = "llama3_70B"
+description = "llama"
+test_template_name = "NeMoLauncher"
+extra_cmd_args = "training.exp_manager.resume_if_exists=false +training.model.nsys_profile.enabled=True +training.model.nsys_profile.start_step=45 training.model.gradient_accumulation_fusion=false +training.model.nsys_profile.end_step=50 +training.model.nsys_profile.gen_shape=False +training.model.nsys_profile.ranks=[0] +training.model.nsys_profile.trace=[nvtx,cuda] training.model.micro_batch_size=1 training.model.virtual_pipeline_model_parallel_size=null training.model.megatron_amp_O2=false training.model.activations_checkpoint_num_layers=null training.model.optim.name=fused_adam training.exp_manager.create_wandb_logger=false training.model.use_cpu_initialization=True +training.model.tokenizer.model=/auto/e2e/israel1/workload_results/llama/tokenizer.model +training.model.fsdp=true ~training.model.optim.bucket_cap_mb ~training.model.optim.overlap_grad_sync ~training.model.optim.overlap_param_sync ~training.model.optim.contiguous_grad_buffer ~training.model.optim.contiguous_param_buffer ~training.model.position_embedding_type"
```

Comment:
We should not reveal our internal path, `/auto/e2e/israel1/workload_results/llama/tokenizer.model`